Src/WitsmlExplorer.Frontend/components/Sidebar/FilterPanel.tsx

```diff
@@ -26,111 +46,128 @@ const FilterPanel = (): React.ReactElement => {
     setCurveThreshold(selectedCurveThreshold);
   }, [selectedCurveThreshold]);
 
+  const wellObjectList = Object.keys(WellboreObjects).map((wellObj: string, index: number) => {
+    return (
+      <EdsProvider key={index}>
+        <Checkbox label={wellObj} style={{ height: "0.75rem" }} defaultChecked key={index} />
```

Comment:
These checkboxes don't do anything. #1592 specifies that "Only the selected well objects are displayed in the wellbore section".

Reply:
Checkboxes of objects that are not yet implemented should be unchecked and disabled. Well and wellbore should be checked and disabled, given that it does not make much sense to toggle them off.

Reply:
updated